### ISSUE TITLE
CMS: Articles from UTSA Today are missing alt text

### ISSUE THREAD

**Issue by lalov1** [utsa-asc/college-dls#478]
![Image](https://user-images.githubusercontent.com/7785251/190688927-b21022ac-2353-4ee2-83a7-b5d8c3affc3b.png)

**Comment by garza-utsa** [comment]
updated the news-group format to use the article headline as a alt text for the photo
